api/src/routes/experiments.ts

```diff
@@ -1,11 +1,13 @@
 import { Router, Response, Request } from "express";
 import { route } from "@fosscord/api";
+import { experiments, with_guild_experiments } from "./experiments.json";
 
 const router = Router();
 
 router.get("/", route({}), (req: Request, res: Response) => {
-	// TODO:
-	res.send({ fingerprint: "", assignments: [] });
+	console.log(req.params)
```

Comment:
Please don't "console.log()" as it can expose some information

Reply:
Forgot to remove that!